app/components/trainees/sort_links/view.html.erb

```diff
@@ -1,7 +1,15 @@
+<div class="app-records-actions">
+<% if FeatureService.enabled?(:trainee_export) %>
```

Comment:
This export button is not a sort link, so it shouldn't be included in the `SortLinks` view component. If should be rendered where the `SortLinks` component is rendered, but not inside it.